scipy/stats/tests/test_continuous_basic.py

```diff
@@ -318,7 +318,7 @@ def test_rvs_broadcast(dist, shape_args):
     shape_only = dist in ['argus', 'betaprime', 'dgamma', 'dweibull',
                           'exponnorm', 'genhyperbolic', 'geninvgauss',
                           'levy_stable', 'nct', 'norminvgauss', 'rice',
-                          'skewnorm', 'semicircular']
+                          'skewnorm', 'semicircular', 'gennorm']
```

Comment:
Can you remind me what this is skipping and what it still tests?

Reply:
It tests that broadcastable loc, scale, and parameter arrays generate correct shaped result but it skips checking if the result itself is consistent with `np.vectorize`d `rvs`. As the `_rvs` method here uses 2 random numbers (one from gamma and one from uniform) to generate one variate, the latter test wouldn't pass because `np.vectorize`d version would call `random_state.gamma` and `random_stats.random` alternatively `size` number of times while the default vectorized `_rvs` method calls them in batch (i.e. samples `size` number of variates from each as a single batch) which generate two different stream of random numbers.

Reply:
OK I figured. That's why you added the test that actually checks the result when there are arrays.